tools/detekt.yml

```diff
@@ -31,6 +31,21 @@ style:
     excludes: [ '**/*.kts' ]
     ignoreAnnotated:
       - 'Preview'
+      - 'PreviewScreens'
+
+complexity:
+  LongMethod:
+    active: true
+    ignoreAnnotated:
+      - 'Preview'
+      - 'PreviewScreens'
+      - 'Composable'
```

Comment:
imo its a more correct way and simpler to suppress it globally instead of writing it for each function :) this way we could get rid of many suppress annotations

Reply:
In the first place, the suppressions should not exist, as we should avoid creating `LongParameterList`, `Complexity`, or `LongMethod` composable functions. But I know sometimes it's hard :) These Detekt rules exist for a reason, and we have aligned with these best practices for a long time. Suppressing it globally, we would not apply it to any part of the project or its future functions. Instead, if we suppress it when we know we can't avoid it, like in several design-lib functions. So let's check if we can suppress it for the design-lib module globally?

Reply:
done